spec/lib/web_of_science/client_spec.rb

```diff
@@ -61,18 +61,12 @@
     context 'when there are no matches returned for SessionID' do
       let(:null_logger) { Logger.new('/dev/null') }
 
-      before do
-        savon.expects(:close_session).returns(no_session_matches)
-        NotificationManager.class_variable_set(:@@wos_logger, nil)
-        allow(Logger).to receive(:new).and_return(null_logger)
-      end
-      it 'works' do
+      before { savon.expects(:close_session).returns(no_session_matches) }
+
+      it 'creates a logger and works' do
+        expect(WebOfScience).to receive(:logger).and_return(null_logger)
         expect(wos_client.session_close).to be_nil
       end
-      it 'creates a logger' do
-        expect(NotificationManager).to receive(:wos_logger).and_call_original
-        wos_client.session_close
-      end
     end
```

Comment:
Best rspec pratices are to have only 1 expectation per spec.  This change is moving the code away from a best practice.  Apart from that, it is consistent with the intention to move the logger method (so that part of it is correct).

There is really only 1 line that needs to change in this spec.  At the previous line 66, the change would be to replace `NotificationManager` with `WebOfScience`.  That is the minimal change that works for the move-method refactor.

Reply:
Read it again.  There is only one expectation.

Reply:
Huh?  The change collapses two specs into one, so it has two expectations, right?